packages/framework-types/src/searcher.ts

```diff
@@ -11,10 +18,10 @@ export type SearcherFunction<TObject> = (className: string, filters: FilterFor<T
  */
 export class Searcher<TObject> {
   // private offset?: number
-  // private limit?: number
+  private maxItems?: number
```

Comment:
I think I know why you call this `maxItems` and not limit (because there was a conflict with the function limit(), correct?)

If so, there is a patter that we are following in other classes like Booster config: using the same name but with an underscore prefix: `_limit`. 

Let me know if you like it.

Reply:
I'm letting you know that I like more that approach and I'm adding the change 😂 thanks!!